transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -116,10 +116,27 @@ static jint netty_epoll_native_timerFd(JNIEnv* env, jclass clazz) {
 }
 
 static void netty_epoll_native_eventFdWrite(JNIEnv* env, jclass clazz, jint fd, jlong value) {
-    jint eventFD = eventfd_write(fd, (eventfd_t) value);
-
-    if (eventFD < 0) {
-        netty_unix_errors_throwChannelExceptionErrorNo(env, "eventfd_write() failed: ", errno);
+    uint64_t val;
+
+    for (;;) {
```

Comment:
is it possible to get into some type of weird busy loop here?

Reply:
nope...

Reply:
ok so eventfd_write isn't going to continuously return -1 and eventfd_read 0.